### PR TITLE
Fix bug in order total shipping rules

### PR DESCRIPTION
**Pull request by loevgaard**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| License         | MIT

The shipping method rules for order totals wasn't working because of this bug.

### CODE REVIEW

**Comment by AdamKasp** [comment]
Hello @loevgaard, yesterday we changed branches on Sylius, and old `master` is now `1.8`, could you rebase your fix with `1.8`? :)

**Comment by loevgaard** [comment]
@AdamKasp: Not sure I did it right. Can you confirm?

File: src/Sylius/Bundle/CoreBundle/Application/Kernel.php [hunk 1]
```diff
@@ -31,13 +31,13 @@
 
 class Kernel extends HttpKernel
 {
-    public const VERSION = '1.8.0-DEV';
+    public const VERSION = '1.9.0-DEV';
```
Comment (AdamKasp) [inline src/Sylius/Bundle/CoreBundle/Application/Kernel.php:34]: Changes in this file should be reverted.

**Review by GSadee — COMMENTED** [review]
Could you adjust `OrderTotalGreaterThanOrEqualRuleCheckerSpec` to this fix?

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Joachim! :1st_place_medal:

**Pull request merged**